### PR TITLE
hcm: small fixes for simulation

### PR DESCRIPTION
**Pull request by SammyRamone**
## Proposed changes
Some small fixes for HCM in simulation.

## Necessary checks
- [x] Update package version
- [x] Run `catkin build`
- [x] Write documentation
- [x] Create issues for future work
- [x] Test on your machine
- [ ] Test on the robot
- [x] Put the PR on our Project board

### CODE REVIEW

File: bitbots_hcm/src/bitbots_hcm/hcm_dsd/decisions/decisions.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def __init__(self, blackboard, dsd, parameters=None):
 
     def perform(self, reevaluate=False):
         self.clear_debug_data()
-        if self.blackboard.simulation_active or self.blackboard.visualization_active:
+        if self.blackboard.visualization_active:
             # we will have no problems with hardware in simulation or visualization
```
Comment (timonegk) [inline bitbots_hcm/src/bitbots_hcm/hcm_dsd/decisions/decisions.py:84]:
```suggestion
            # we will have no problems with hardware in visualization
```

File: bitbots_hcm/src/bitbots_hcm/hcm_dsd/hcm_blackboard.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def __init__(self):
         self.pressure_sensors_installed = rospy.get_param("hcm/pressure_sensors_installed", False)
         self.pressure_timeout_duration = rospy.get_param("hcm/pressure_timeout_duration")
         self.last_pressure_update_time = None
-        self.pressures = [0] * 8
+        self.pressures = [100] * 8
```
Comment (timonegk) [inline bitbots_hcm/src/bitbots_hcm/hcm_dsd/hcm_blackboard.py:66]: Why is this changed?

Comment (SammyRamone) [reply bitbots_hcm/src/bitbots_hcm/hcm_dsd/hcm_blackboard.py:66]: otherwise the HCM thinks its picked up (no forces on the feet) till it receives the first foot pressure message. in webots we have no pressure sensors yet, so the HCM would always stay picked up. on real robot this is also a possible race condition which could make the HCM go in the wrong state during boot

Comment (timonegk) [reply bitbots_hcm/src/bitbots_hcm/hcm_dsd/hcm_blackboard.py:66]: Okay, please add a comment.

**Review by timonegk — APPROVED** [review]
